### ISSUE TITLE
Manual Play Pickit (potions)

### ISSUE THREAD

**Issue by Heycastortroy** [kolton/d2bot-with-kolbot#2957]
I'm having some problems trying to get pickit working on manual play (botmap.dbj or blank.dbj). I read all of the issues on here and I have not been able to get pickit to work properly. At one point it was auto picking up items from the kolton.nip file but it will not pick up pots. 

I am using the newest revision.

I've spent about 5+ hours trying to get this to work if anyone can share some words of wisdom I would greatly appreciate it!

**Comment by gtoilet** [comment]
https://github.com/kolton/d2bot-with-kolbot/issues/1445

**Comment by Heycastortroy** [comment]
> #1445

I have already tried that. The pickit works but it will not pick up pots.

**Comment by gtoilet** [comment]
you added pots to pickit?

**Comment by Heycastortroy** [comment]
> you added pots to pickit?

Yes. Kolton.nip has uncommented pots by default. I have also tried using a .nip file with just pots listed. 

Pots are picked up just fine in scripted play.
